spec.bs

```diff
@@ -4336,24 +4337,31 @@ A <dfn>per participant metrics</dfn> is a [=struct=] with the following [=struct
 representing metrics aggregated over a particular participant (e.g. bidder or seller). It has:
 <dl dfn-for="per participant metrics">
   : <dfn>participating interest group count</dfn>
-  :: A {{long}}, initially 0.
+  :: A {{long}}, initially 0. See "<code>[=signal base value/participating-ig-count=]</code>".
   : <dfn>code fetch time averager</dfn>
-  :: An [=averager=].
+  :: An [=averager=]. Used to compute "<code>[=signal base value/average-code-fetch-time=]</code>".
   : <dfn>script timeouts occurred</dfn>
-  :: A {{long}}, initially 0.
+  :: A {{long}}, initially 0. Used to compute
+    "<code>[=signal base value/percent-scripts-timeout=]</code>".
   : <dfn>cumulative timeouts occurred</dfn>
   :: A {{long}}, initially 0. Number of interest groups affected by the cumulative timeout
-    happening.
+    happening. Used to compute "<code>[=signal base value/percent-igs-cumulative-timeout=]</code>".
   : <dfn>cumulative buyer time</dfn>
   :: A [=duration=] in milliseconds, initially 0.
+    See "<code>[=signal base value/cumulative-buyer-time=]</code>".
   : <dfn>script executions attempted</dfn>
-  :: A {{long}}, initially 0.
+  :: A {{long}}, initially 0. The denominator for computation of

```

Comment:
probably "for computation of some percentage metrics, such as ...". So that we don't need to update it everytime we add new percentage metrics.

Reply:
Done